src/libraries/System.DirectoryServices/ref/System.DirectoryServices.manual.cs

```diff
@@ -9,4 +9,10 @@ namespace System.DirectoryServices
     [System.ComponentModel.TypeConverter(typeof(DirectoryEntryConverter))]
     public partial class DirectoryEntry { }
     internal class DirectoryEntryConverter { }
+
+    [System.ObsoleteAttribute("Code Access Security is not supported or honored by the runtime.", DiagnosticId = "SYSLIB0003", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
```

Comment:
Is this in the .manual.cs file because the ref auto-generation is choking on the locally defined attribute?

Reply:
would it make more sense to move this attribute back to the System.DirectoryServices.cs file instead so that the whole class is defined only in one place? Or on the other hand, should we move the full definition of DirectoryServicesPermission to the manual.cs file?

Reply:
The tool wasn't producing the attribute for me.  I'll apply it within the `System.DirectoryServices.cs` file though--that's the approach I took on all the other obsoletions.